ruleset/rules/0755-office365_rules.xml

```diff
@@ -0,0 +1,928 @@
+    </rule>
+
+    <rule id="91589" level="5">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^61$</field>
+        <description>Office 365: Events related to compromised user alerts.</description>
+        <options>no_full_log</options>
+        <group>InformationWorkerProtection</group>
+    </rule>
+
+    <rule id="91590" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^62$</field>
+        <description>Office 365: Email campaign events from Microsoft Defender for Office 365.</description>
+        <options>no_full_log</options>
+        <group>Campaign</group>
+    </rule>
+
+    <rule id="91591" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^63$</field>
+        <description>Office 365: Endpoint DLP events.</description>
+        <options>no_full_log</options>
+        <group>DLPEndpoint</group>
+    </rule>
+
+    <rule id="91592" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^64$</field>
+        <description>Office 365: Automated incident response (AIR) events.</description>
+        <options>no_full_log</options>
+        <group>AirInvestigation</group>
+    </rule>
+
+    <rule id="91593" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^65$</field>
+        <description>Office 365: Quarantine events.</description>
+        <options>no_full_log</options>
+        <group>Quarantine</group>
+    </rule>
+
+    <rule id="91594" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^66$</field>
+        <description>Office 365: Microsoft Forms events.</description>
+        <options>no_full_log</options>
+        <group>MicrosoftForms</group>
+    </rule>
+
+    <rule id="91595" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^67$</field>
+        <description>Office 365: Application audit events.</description>
+        <options>no_full_log</options>
+        <group>ApplicationAudit</group>
+    </rule>
+
+    <rule id="91596" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^68$</field>
+        <description>Office 365: Events tracked by the Communication compliance offensive language model.</description>
+        <options>no_full_log</options>
+        <group>ComplianceSupervisionExchange</group>
+    </rule>
+
+    <rule id="91597" level="5">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^69$</field>
+        <description>Office 365: Events related to the customer key encryption service.</description>
+        <options>no_full_log</options>
+        <group>CustomerKeyServiceEncryption</group>
+    </rule>
+
+    <rule id="91598" level="3">
```

Comment:
I think this level should be higher since these alerts include sensitive labels.

Reply:
Done